CONDUCT.md

```diff
@@ -0,0 +1,9 @@
+# Contributing To The Project
+
+If you decide to contribute anything, please do the following:
+
+1. Fork this repository
+1. Create a new branch from the latest `develop` (read more [here](https://guides.github.com/introduction/flow/))
```

Comment:
```suggestion
1. Create a new branch from the latest `master` (read more [here](https://guides.github.com/introduction/flow/))
```
We don't have a `develop` branch, but we do have a `master` branch.

Reply:
Sure, good call!